server/core/templates/oauth2_access_denied.html

```diff
@@ -0,0 +1,16 @@
+(% extends "base.html" %)
+
+(% block title %)Oauth2 Access Denied(% endblock %)
```

Comment:
We don't need to mention it's OAuth2 on this one
```suggestion
(% block title %)Access Denied(% endblock %)
```